cli/popper/commands/cmd_run.py

```diff
@@ -512,12 +518,19 @@ def run_pipeline(project_root, pipe_n, pipe_d, env, timeout,
             status = run_on_host(project_root, pipe_n, pipe_d, skip_list,
                                  timeout_parsed, output_dir)
         elif env != 'host':
-            status = \
-                run_in_docker(project_root, pipe_n, pipe_d, env,
-                              timeout, skip, ignore_errors, '{}/{}/{}'.
-                              format(output_dir, env.replace('/', '_'),
-                                     number_of_run if env_vars else ""),
-                              env_vars)
+            for idx, arg in enumerate(args if args else [""]):
+
+                idx = str(idx) if len(args) > 1 else ""
+                conn = "_" if len(args) > 1 else ""
+                idx = str(number_of_run) + conn + idx if env_vars else idx
```

Comment:
Looks like this is running the pipeline, once for every different argument, right? I think what we would like is to just run it once, and pass the entire list to the docker daemon

Reply:
The idea I had in mind is that you can execute multiple sets of arguments. For example, something like: 

```bash
envs:
      debian-9:
        args: 
        - - --runtime=runc
           - --ipc=host
       - - --runtime=nvidia
         - --ipc=private
```
This will execute for --runtime=runc,--ipc=host and for --runtime=nvidia,--ipc=private. This would be useful if you want to test using different arguments for docker in the same environment.

Reply:
Oh, I see. Should be okay now.